src/app/data-planes/views/DataPlaneDetailView.vue

```diff
@@ -311,6 +463,24 @@ const warnings = computed(() => {
 </script>
 
 <style lang="scss" scoped>
+.traffic {
+  padding: 0;
+  container-type: inline-size;
+  container-name: traffic;
+  .columns {
```

Comment:
**Suggestion**: Please use a different class name to avoid the conflict with the global `columns` styles.

Reply:
🤔 This is `.traffic .columns` within a scoped block so I doubt it will conflict with anything?

The only way would be if we were to put more columns inside `.traffic` but specifically in this file. Considering the size of the columns here already I'm not sure if that would happen, and if in the extremely weird case it did we would be editing this file, so we could just update the selector then?

I dunno, did I miss something there?

On another note: just speaking out loud more than anything, I'm unsure how Vue would deal with `container-name`s within `scoped` I would assume that aren't scoped.

Reply:
> 🤔 This is .traffic .columns within a scoped block so I doubt it will conflict with anything?

Not in that direction, no, but if you add a use of `columns` inside of `.traffic` that doesn’t want these new styles, you have a conflict.

Just a good practice to avoid this problem from even being possible by not re-using global class names like that.

> just speaking out loud more than anything, I'm unsure how Vue would deal with `container-name`s within `scoped` I would assume that aren't scoped.

Scoping is done on the level of selector lists and property declarations (i.e. `container-name: traffic) and at rules (i.e. `@container`) should be affected by it at all.

Reply:
> Not in that direction, no, but if you add a use of columns inside of .traffic that doesn’t want these new styles, you have a conflict.

> > The only way would be if we were to put more columns inside .traffic but specifically in this file. Considering the size of the columns here already I'm not sure if that would happen, and if in the extremely weird case it did we would be editing this file, so we could just update the selector then?

Reply:
Still, avoiding the clash proactively is the right thing to do because you completely remove the need to even remember that there can be a clash.

Reply:
> Scoping is done on the level of selector lists and property declarations (i.e. container-name: traffic) and at rules (i.e. @container`) should be affected by it at all.

Was that a typo as it doesn't look like container-names get scoped? I guess you meant that it's not affected?

![Screenshot 2023-12-06 at 13 06 44](https://github.com/kumahq/kuma-gui/assets/554604/845ff7e7-d9d1-4ea6-a29b-6421a19fa259)

Reply:
> and if in the extremely weird case it did we would be editing this file, so we could just update the selector then?

Was this just a suggestion or is this a blocker?

Reply:
> Was that a typo as it doesn't look like container-names get scoped? I guess you meant that it's not affected?

Yes, that was a typo, it should read "shouldn’t be".

> Was your original comment just a suggestion or is this a blocker?

Just a suggestion. One I thought was a straightforward improvement and strictly better.